tests/std/tests/GH_000108_deque_incomplete_value_types/test.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <assert.h>
+#include <deque>
+
+using namespace std;
+
+void test_gh_000108_push_pop_size();

```

Comment:
Is there a reason for the separate declaration and definition of that function? We conventionally only define it before the main function

Reply:
I partially duplicated the test.cpp file from that of Dev10_860421.

To be honest I'm not familiar with the tests of MSVC STL and don't know what are expected to be tested...